packages/gatsby-source-wordpress/README.md

```diff
@@ -83,6 +83,11 @@ plugins: [
       // Set verboseOutput to true to display a verbose output on `npm run develop` or `npm run build`
       // It can help you debug specific API Endpoints problems
       verboseOutput: false,
+      // Search and Replace Urls across WordPress content
+      searchAndReplaceURLs: {
```

Comment:
Sorry for changing my mind again but how about we name the option & function `searchAndReplaceContentUrls`? That's super clear.

Reply:
Personally I think the comment above makes it clear enough, don't you think?